server/index.js

```diff
@@ -1,33 +1,11 @@
-const express = require('express');
-const app = express();
-const bodyParser = require('body-parser');
+const app = require('./app.js');
```

Comment:
Ah, I see what you were saying with how you had to split up this file in order for jest tests to work.